src/test/java/com/easyrest/tests/Administrator/AdministratorTest.java

```diff
@@ -0,0 +1,85 @@
+package com.easyrest.tests.Administrator;
+
+import com.easyrest.facade.AdministratorOperationsFacade;
+import com.easyrest.facade.AuthorizedHeaderMenuPanelFacade;
+import com.easyrest.facade.SignInFacade;
+import com.easyrest.pages.SignInPage;
+import com.easyrest.tests.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.easyrest.config.ConfigProvider.administratorEmail;
+import static com.easyrest.config.ConfigProvider.administratorPassword;
+import static com.easyrest.constants.Constants.AdministratorTabMenu.*;
+
+public class AdministratorTest extends BaseTest {
+    private AdministratorOperationsFacade administrator;
+
+    @BeforeMethod
```

Comment:
why do you need to login and log out after each test?

Reply:
each test should be independent of any other test.

Reply:
> each test should be independent of any other test.

but if first test fails the program will execute second test. If you do login and log out after each test there is more time execution.